### PR TITLE
Fix : issue #1038 and #1040

### LINKED ISSUE #1038: UI: versions shows as 0.3 but when we expand the versions it only show up to 0.2
image

image

### LINKED ISSUE #1040: UI: versions breaks the welcome link when open
If I click on the "welcome link" nothing happens.
Similarly for the link "Need Help"

image
image
image

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
Fixes #1038 
Fixes #1040 

### Describe your changes :

I worked on issue #1038 #1040 

#
### Type of change :

- [x] Bug fix

#
### Frontend Preview (Screenshots) :
Fix for #1038 
image

Fix for #1040 
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@shahsank3t, @darth-coder00

### CODE REVIEW

File: catalog-rest-service/src/main/resources/ui/src/components/SchemaTab/SchemaTab.component.tsx [hunk 1]
```diff
@@ -128,7 +128,7 @@ const SchemaTab: FunctionComponent<Props> = ({
       </div>
       <div className="row">
         <div className="col-sm-12">
-          {checkedValue === 'schema' ? (
+          {checkedValue === 'schema' && columns?.length ? (
```
Comment (darth-coder00) [inline catalog-rest-service/src/main/resources/ui/src/components/SchemaTab/SchemaTab.component.tsx:131]: How will it look if there are no columns?

Comment (darth-coder00) [reply catalog-rest-service/src/main/resources/ui/src/components/SchemaTab/SchemaTab.component.tsx:131]: And not sample-data as well

Comment (Sachin-chaurasiya) [reply catalog-rest-service/src/main/resources/ui/src/components/SchemaTab/SchemaTab.component.tsx:131]: If there are no columns then it won't render the EntityTable.

Comment (Sachin-chaurasiya) [reply catalog-rest-service/src/main/resources/ui/src/components/SchemaTab/SchemaTab.component.tsx:131]: > And not sample-data as well

if there is no sample data then it won't render the sample data table.

**Review by darth-coder00 — APPROVED** [review]
LGTM

**Pull request merged**